mirage/config.js

```diff
@@ -9,18 +9,18 @@ import { Response } from 'mirage-server';
  * @returns {Function} route handler for a search route of this
  * resource type
  */
-function searchRouteFor(resourceType, filter = () => true) {
+function searchRouteFor(resourceType, filter = (title, req) => {
+  let query = req.queryParams.q.toLowerCase();
+  return title.name && title.name.toLowerCase().includes(query.toLowerCase());
```

Comment:
To avoid confusion, we should use `model` here instead of `title` since this applies to packages and providers.

Also `let query` already has `toLowerCase` so the second `query.toLowerCase()` isn't necessary.